nasa-gibs/worldview
# WV-2359 Disable Set Area of Interest Without Selection

## Description

Fixes # .

Edited containers/sidebar/smart-handoff.js to check for a selected collection before enabling the earthdata button and rendering the set area of interest button.

## How To Test

When visiting this URL 

http://localhost:3000/?v=-157.50605321419718,-124.98135679600885,138.05460701253722,86.99656135638496&sh=VIIRS_SNPP_CorrectedReflectance_TrueColor,C1344298008-LANCEMODIS&t=2021-10-18-T12%3A27%3A23Z

The download earthdata button should be disabled and the set area of interest button should be hidden until you select a collection.
